accessibility-oscilloscope/overlay/usr/bin/wacom-btn

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
-libinput record /dev/input/event1 | awk '/BTN_[0-3]\s*0/ { print substr($11,5,1) }'
+DEVICE=$(libinput list-devices | grep "Wacom Intuos BT M Pad" -A1 | awk '/Kernel:/ {print $2}')
```

Comment:
Actually for this we might want to continue to hardcode the DEVICE at least since libinput list-devices still has the following error:
Maybe just DEVICE=/dev/input/event1
```
# libinput list-devices
event1  - Wacom Intuos BT M Pad: libinput bug: missing tablet capabilities: pen resolution. Ignoring this device.
Device:           Wacom Intuos BT M Pen
Kernel:           /dev/input/event0
Group:            1
Seat:             seat0, default
Size:             216x135mm
Capabilities:     tablet
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   none
Rotation:         n/a
```